### PR TITLE
func_802BD200_31B220

### PR DESCRIPTION
**Pull request by Ponmander**

### CODE REVIEW

File: src/world/partner/kooper.c [hunk 1]
```diff
@@ -8,7 +8,16 @@ INCLUDE_ASM(s32, "world/partner/kooper", func_802BD144_31B164);
 
 INCLUDE_ASM(s32, "world/partner/kooper", func_802BD17C_31B19C);
 
-INCLUDE_ASM(s32, "world/partner/kooper", func_802BD200_31B220);
+//INCLUDE_ASM(s32, "world/partner/kooper", func_802BD200_31B220);
+
+extern s32 D_802BEC54;
```
Comment (ethteck) [inline src/world/partner/kooper.c:13]: Externs are typically put at the top of the file with eachother. This one would go before the unkPartnerStruct* one at the very top, to put them in ram order.

File: src/world/partner/kooper.c [hunk 2]
```diff
@@ -8,7 +8,16 @@ INCLUDE_ASM(s32, "world/partner/kooper", func_802BD144_31B164);
 
 INCLUDE_ASM(s32, "world/partner/kooper", func_802BD17C_31B19C);
 
-INCLUDE_ASM(s32, "world/partner/kooper", func_802BD200_31B220);
+//INCLUDE_ASM(s32, "world/partner/kooper", func_802BD200_31B220);
```
Comment (ethteck) [inline src/world/partner/kooper.c:11]: This comment can be removed

File: src/world/partner/kooper.c [hunk 3]
```diff
@@ -8,7 +8,16 @@ INCLUDE_ASM(s32, "world/partner/kooper", func_802BD144_31B164);
...
+
+extern s32 D_802BEC54;
+
+void func_802BD200_31B220(Npc* npc) {
+    npc->collisionHeight = 0x25;
```
Comment (ethteck) [inline src/world/partner/kooper.c:16]: The height and radius values should be written in decimal

**Review by ethteck — CHANGES_REQUESTED** [review]
Nice job! One more thing: it makes more sense to name this PR the name of the function you decomped, instead of the file it came from. If you were doing more functions in this file, then the current name would make more sense. but since it's just one, better to be specific. you can rename the PR with the "edit" button

**Comment by Brotenko** [comment]
Nothing to add besides Ethan's comments! Just make sure to run ./coverage.py before you push the requested changes and you should be golden - Good job!

**Review by nanaian — COMMENTED** [review]
Nothing to add, gj

**Comment by ethteck** [comment]
I made the requested changes and am merging them since Ponderman said he's too busy to currently

**Comment by ethteck** [comment]
Nevermind, this is already done by Rain

**Pull request closed without merging**